polling_stations/apps/data_importers/data_quality_report.py

```diff
@@ -247,14 +249,15 @@ def get_districts_containing_more_stations(self):
 class AddressReport:
     def __init__(self, council_id):
         self.council_id = council_id
+        self.gss_code = Council.objects.get(pk=council_id).geography.gss
```

Comment:
I guess `gss_code` could be a property on the council model, but part of the point of this pr is to make it a seperate concept - so I'm inclined not to.